kuma/payments/forms.py

```diff
@@ -137,3 +197,137 @@ def make_charge(self):
                     )
                 )
         return False
+
+
+class RecurringPaymentForm(ContributionForm):
+
+    def __init__(self, *args, **kwargs):
+        super(RecurringPaymentForm, self).__init__(*args, **kwargs)
+        self.fields['donation_choices'].choices = RECURRING_PAYMENT__CHOICES
+
+    @staticmethod
+    def create_customer(email, token, user, name):
+        customer = stripe.Customer.create(
+            email=email,
+            source=token,
+            metadata={'name': name}
+        )
+        user.stripe_customer_id = customer.id
+        user.save()
+        return customer.id
+
+    @staticmethod
+    def update_source_name(source_id, name):
+        """Updates the source name with the users defined name"""
+        source = stripe.Source.retrieve(source_id)
+        source.owner["name"] = name
+        source.save()
+
+    def make_recurring_payment_charge(self, user, update_source_name=False):
+        amount = self.get_amount()
+        token = self.cleaned_data.get('stripe_token', '')
+        try:
+            if token and amount:
+                if user.stripe_customer_id:
+                    # ensure that customer is active
+                    customer = stripe.Customer.retrieve(user.stripe_customer_id)
```

Comment:
I created a customer on the Mozilla stripe account, and then switched to my personal account to create a test product. I got this error: 

> kuma.payments.forms:ERROR Stripe: Invalid parameters were supplied to Stripe API: <my name> [<my email>] Request req_RequestCode: No such customer: cus_CustomerCode

I fixed it by clearing the ``stripe_customer_id`` and trying again.

I don't think any change is required - I'm not sure what we should do in this case, and it feels like only developers will encounter it, unless we change Stripe accounts in the future.